src/arrays.js

```diff
@@ -1,62 +1,70 @@
 const getNthElement = (index, array) => {
-  // your code here
+  while (index>=array.length){
+    index = index - array.length
+  }
+  return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString()
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',')
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element)
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element)
+ 
+
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1)
+
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.join().split(',')
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map((word)=>word.toUpperCase())
```

Comment:
Would be really good to get your IDE set up to automatically format your work on save, as it would make things easier to read and save you some extra typing to boot. In VSCode, if you go into your settings and search for 'format on save', turn this on (in both your workspace settings AND user settings), when you save your file it will automatically format it for you so you don't even have to remember to do it!
This is what this would look like after doing that:
```
  return strings.map(word => word.toUpperCase());
```
Rather than comment similar throughout your work here, Ill just point out logic or other non-formatting type issues as if you were to do this with your project, it will fix all the extra whitespace/not enough spacing/unnecessary syntax for you!